checks/raw/pinned_dependencies_test.go

```diff
@@ -213,6 +218,11 @@ func TestNonGithubWorkflowPinning(t *testing.T) {
 			filename: "./testdata/.github/workflows/workflow-mix-pinned-and-non-pinned-non-github.yaml",
 			warns:    1,
 		},
+		{
+			name:     "Can't detect OS, but still detects unpinned Actions",
+			filename: "./testdata/.github/workflows/github-workflow-unknown-os.yaml",
+			warns:    2, // 1 in job with unknown OS, 1 in job with known OS
```

Comment:
do these warn numbers (or the comments) needed updated since the statement is an info?